.circleci/config.yml

```diff
@@ -9144,6 +9012,101 @@ workflows:
                 - postnightly
           executor: windows-with-nvidia-gpu
     when: << pipeline.parameters.run_build >>
+  scheduled-ci:
+    triggers:
+      - schedule:
+          cron: "45 0,4,8,12,16,20 * * *"
```

Comment:
`nit`: Can you describe what this cron equals out to in a comment so we have a good view of the frequency?